src/Sylius/Bundle/SettingsBundle/Resources/config/doctrine/model/Settings.orm.xml

```diff
@@ -13,18 +13,18 @@
 
 <doctrine-mapping xmlns="http://doctrine-project.org/schemas/orm/doctrine-mapping">
 
-    <mapped-superclass name="Sylius\Bundle\SettingsBundle\Model\Parameter" table="sylius_settings_parameter">
+    <mapped-superclass name="Sylius\Bundle\SettingsBundle\Model\Settings" table="sylius_settings">
+        <unique-constraints>
+            <unique-constraint columns="schema_alias,namespace"/>
+        </unique-constraints>
+
         <id name="id" column="id" type="integer">
-            <generator strategy="AUTO" />
+            <generator strategy="AUTO"/>
         </id>
 
-        <field name="namespace" column="namespace" type="string" />
-        <field name="name" column="name" type="string" />
-        <field name="value" column="value" type="object" nullable="true" />
-
-        <unique-constraints>
-            <unique-constraint columns="namespace,name" name="name_idx" />
-        </unique-constraints>
+        <field name="schemaAlias" column="schema_alias" type="string" nullable="false"/>
+        <field name="namespace" type="string" nullable="true"/>
```

Comment:
Yes, this is for global settings, basically I switched out the schema alias with what previously was `namespace`. The `namespace` now is just used to isolate settings so you can, as the name says, namespace them. Is this wrong?